File: Python Excel Examples/WorkbookApiDemo/createWorkbook.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

# Available formats: xlsx, xls, xlsm, xltm, xltx, ods
name = "CreateWorkbook.xlsx"
password = ""
inputPassword = ""
storage = ""
data="D:/data/test.xlsx"
folder = "ExcelDocument"
api.create_workbook(name, data=data, input_password=inputPassword, password=password, storage=storage,folder=folder)

File: PHP Excel Examples/WorkbookApiDemo/createWorkbook.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorkbookApi($configuration);

#supports formats: Xlsx, Xlsb, Xls,xlsm,xltm,xltx, Ods
$data = "D:/data/test.xlsx";
$file = new SplFileObject($data);;
#supports formats: Xlsx, Xlsb, Xls,xlsm,xltm,xltx, Ods
$name = "CreateWorkbook.xlsx";
$inputPassword = "Password";
$password = "123";
$storage = null;
$folder = "output";
$result = $apiInstance->CreateWorkbook($name, $file, $inputPassword, $password, $storage, $folder);